ui/components/component-library/modal-content/modal-content.scss

```diff
@@ -6,6 +6,22 @@
   overflow: auto;
   overscroll-behavior-y: none;
 
+  // Scroll bar styles
+  // Firefox
+  scrollbar-width: thin;
```

Comment:
## Firefox
Scroll bar style in firefox

### Before
<img width="378" alt="Screenshot 2023-09-05 at 1 16 35 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/9f9d10ed-52a2-4e61-95ab-1662794273f1">
<img width="958" alt="Screenshot 2023-09-05 at 1 17 03 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/1a771303-f06a-4c01-b0a2-1e22268996b1">

### After

<img width="991" alt="Screenshot 2023-09-05 at 1 18 59 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/6db86d0f-f783-443e-b145-8c86412b4ab1">
<img width="375" alt="Screenshot 2023-09-05 at 1 19 30 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/f03a7534-3420-4ea6-86cb-909a0167edf9">